src/VideoJS.as

```diff
@@ -250,6 +250,12 @@ package{
                 case "duration":
                     return _app.model.duration;
                     break;
+                case "seekStart":
```

Comment:
I think we should name these `seekableStart` and `seekableEnd` to avoid any confusion with seek actions

Reply:
It also might be good to add a comment here, (or within the other providers) mentioning that these values are only meant to be used with the HLSProvider